app/utils/live-paginated-collection.js

```diff
@@ -106,4 +106,8 @@ LivePaginatedCollection.reopenClass({
   }
 });
 
+LivePaginatedCollection.toString = function () {
+  return 'LivePaginatedCollection';
+};
+
```

Comment:
Is this for debugging help?

Reply:
For tests actually, but I guess it's good to have anyway (and maybe I'll start adding it more?). With this you get something like `<LivePaginationCollection:ember1213>` instead of `<subclass of (ArrayProxy):ember1213` when you do `toString()` on an instance.

Reply:
I'm checking it here: https://github.com/travis-ci/travis-web/pull/1403/files#diff-8916b1cc56744b82d8052735c3777361R242

Reply:
oh yes, makes sense, I missed that 😬